client/flutter/divkit/lib/src/core/widgets/input/div_input_model.dart

```diff
@@ -87,6 +90,12 @@ class DivInputModel with EquatableMixin {
       final keyboardType = await data.keyboardType.resolveValue(
         context: context,
       );
+
+      final inputFormatters = data.inputFormatters != null
```

Comment:
Here you can apply the formatter with the desired `keyboardType`.

It will look something like this and it seems to also perform the functionality:
```dart
final keyboardType = (await data.keyboardType.resolveValue(context: context)).map(
  singleLineText: () => TextInputType.text,
  multiLineText: () => TextInputType.multiline,
  phone: () => TextInputType.phone,
  number: () => TextInputType.number,
  email: () => TextInputType.emailAddress,
  uri: () => TextInputType.url,
  password: () => null,
);

final inputFormatters = keyboardType == TextInputType.number
          ? [FilteringTextInputFormatter.digitsOnly]
          :  [];
```